R-package/R/lstm.R

```diff
@@ -0,0 +1,589 @@
+require(mxnet)
+
+# lstm cell symbol
+lstm <- function(num.hidden, indata, prev.state, param, seqidx, layeridx, dropout=0) {
```

Comment:
add some roxygen comments for new functions in R/
and then run `roxygen2::roxygenise()` to regenerate the Rd files